FreeRTOS-Plus/Demo/corePKCS11_Windows_Simulator/WIN32.vcxproj

```diff
@@ -0,0 +1,357 @@
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\library\x509write_csr.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\library\x509_create.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\library\x509_crl.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\library\x509_crt.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\library\x509_csr.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\library\xtea.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls_utils\mbedtls_error.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls_utils\mbedtls_utils.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\source\iot_pkcs11.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\source\iot_pki_utils.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\source\portable\mbedtls\iot_pkcs11_mbedtls.c" />
+    <ClCompile Include="..\..\Source\Application-Protocols/corePKCS11\source\portable\windows\iot_pkcs11_pal.c" />
+    <ClCompile Include="examples\demo_helpers.c" />
+    <ClCompile Include="examples\management_and_rng.c" />
+    <ClCompile Include="examples\objects.c" />
+    <ClCompile Include="examples\mechanisms_and_digests.c" />
+    <ClCompile Include="examples\sign_and_verify.c" />
+    <ClCompile Include="main.c" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\event_groups.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\FreeRTOS.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\portable.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\projdefs.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\queue.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\semphr.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\stack_macros.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\task.h" />
+    <ClInclude Include="..\..\..\FreeRTOS\Source\include\timers.h" />
+    <ClInclude Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\include\mbedtls\aes.h" />
+    <ClInclude Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\include\mbedtls\aesni.h" />
+    <ClInclude Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\include\mbedtls\arc4.h" />
+    <ClInclude Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\include\mbedtls\aria.h" />
+    <ClInclude Include="..\..\Source\Application-Protocols/corePKCS11\3rdparty\mbedtls\include\mbedtls\asn1.h" />

```

Comment:
Forward slashes and backslashes are mixed here

Reply:
Fixed